tests/errmsgs/tyield_view_tuple.nim

```diff
@@ -0,0 +1,22 @@
+discard """
+  nimoutFormat: sexp
+  cmd: "nim check --msgFormat=sexp --filenames=canonical --hints:off $options $file"
+  action: reject
+"""
+
+proc make(x: var int): (int, var int) =
+  (0, x)
+
+iterator iter(): (int, var int) =
+  var x = 0
+  yield make(x) #[tt.Error
```

Comment:
# this code fails to compile because the expression is not a tuple constructor literal